README.md

```diff
@@ -1 +1,28 @@
-# Mobile-Menu
\ No newline at end of file
+
+# Project Name:portfolio
+This is my first project at Microverse.
+- [link is here](https://haftamudesta.github.io/desta451616-hotmail.com.github.io/portfolio.html)
```

Comment:
- [ **OPTIONAL** ] Kindly make sure to add the deployment link in its appropriate section(**## Live Demo**) as on the **[Microverse README template](https://github.com/microverseinc/readme-template)**. I know you can do that.